sklearn/decomposition/tests/test_dict_learning.py

```diff
@@ -575,6 +578,31 @@ def test_sparse_coder_n_features_in():
     assert sc.n_features_in_ == d.shape[1]
 
 
+def test_update_dict():
```

Comment:
```suggestion
def test_update_dict():
    """Non-regression test for issue #4866."""
```

Reply:
We don't use docstrings for the tests, only comments. I added that as a comment.

Reply:
Actually, I think @thomasjpfan mentioned elsewhere that we were transitioning to docstring even in tests now that we no longer use nose (I think this is the reason).

No strong opinion for this PR.

Reply:
> we were transitioning to docstring even in tests now that we no longer use nose (I think this is the reason).

Exactly. Since that we don't make a strong statement, this is still depending on who is reviewing the PR :)